beeai/agents/backport_agent.py

```diff
@@ -70,6 +76,31 @@ def input_schema(self) -> type[TInputSchema]:
     def output_schema(self) -> type[TOutputSchema]:
         return OutputSchema
 
+    def _render_prompt(self, input: TInputSchema) -> str:
+        # Define template function that can be called from the template
+        def backport_git_steps(data: dict) -> str:
+            return get_backport_git_steps(
+                package=data.get("package", ""),
+                jira_issue=data.get("jira_issue", ""),
+                git_url=data.get("git_url", ""),
+                gitlab_user=data.get("gitlab_user", ""),
+                git_user=data.get("git_user", ""),
+                git_email=data.get("git_email", ""),
+                dist_git_branch=data.get("dist_git_branch", ""),
+                dry_run=data.get("dry_run", False)
+            )
```

Comment:
Wouldn't it be possible to pass `data` to `get_*_git_steps` directly (or as an `InputSchema` instance -`InputSchema.model_validate(data)`)?